libraries/localization-utilities/src/TypingsGenerator.ts

```diff
@@ -62,7 +89,31 @@ export class TypingsGenerator extends StringValuesTypingsGenerator {
           });
         }
 
-        return { typings };
+        let exportAsDefaultInterfaceName: string | undefined;
+        if (inferDefaultExportInterfaceNameFromFilename) {
+          const lastSlashIndex: number = Math.max(filePath.lastIndexOf('/'), filePath.lastIndexOf('\\'));
+          let extensionIndex: number = filePath.lastIndexOf('.');
```

Comment:
What if this doesn't find an extension? Or is that not a scenario that this would ever encounter?